great-idea/services.html

```diff
@@ -16,91 +16,116 @@
 
 <body>
 
+  <nav class="services-navigation">
+    <h3 class="nav-items">Services</h3>
+    <h3 class="nav-items">Product</h3>
+    <h3 class="nav-items">Vision</h3>
+    <h3 class="nav-items">Features</h3>
+    <h3 class="nav-items">About</h3>
+    <h3 class="nav-items">Contact</h3>
+    <img src="img/logo.png" alt="Great Idea! Company logo.">
+  </nav>
 
-Services
-Product
-Vision
-Features
-About
-Contact
 
-<img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  <section class="services-container">
```

Comment:
This appears to be a header and this text along with the nav should be within a `header` element:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header